### PR TITLE
[MENFORCER-301] check dependencyManagement

### PR DESCRIPTION
**Pull request by kudrevatykh**
really check dependencyManagement section

### CODE REVIEW

**Comment by kudrevatykh** [comment]
added tests and rebased on master

File: maven-enforcer-plugin/src/it/projects/ban-duplicate-dependencies-versions/pom.xml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+<project>
+    <modelVersion>4.0.0</modelVersion>
```
Comment (elharo) [inline maven-enforcer-plugin/src/it/projects/ban-duplicate-dependencies-versions/pom.xml:23]: nit: maven style for XML files is two space indents:

Comment (kudrevatykh) [reply maven-enforcer-plugin/src/it/projects/ban-duplicate-dependencies-versions/pom.xml]: fixed indent

**Comment by pzygielo** [comment]
I see for new IT that build fails with
```
Found 1 duplicate dependency declarations in this project
```
as expected.
But could it be **verified**, to exclude possible future failures due to **other** than new rule violation reasons? (For example adding _unrecognised tag_ in pom.xml also results in IT build failure which is incorrectly recognized as passed test.)

**Comment by kudrevatykh** [comment]
> I see for new IT that build fails with
> 
> ```
> Found 1 duplicate dependency declarations in this project
> ```
> 
> as expected.
> But could it be **verified**, to exclude possible future failures due to **other** than new rule violation reasons? (For example adding _unrecognised tag_ in pom.xml also results in IT build failure which is incorrectly recognized as passed test.)

added verification script

**Comment by jira-importer** [comment]
Resolve #778

**Pull request merged**